src/vectors/vectorWindow.component.test.tsx

```diff
@@ -65,6 +81,41 @@ describe('Vector Window component', () => {
     screen.getByLabelText('Vector loading');
   });
 
+  it('show control panel button is visible and interactive', async () => {
+    const user = userEvent.setup();
+    const ref = React.createRef<WindowPortal>();
+
+    Object.defineProperty(ref, 'current', {
+      value: {
+        getWindow: vi.fn(() => ({
+          Plotly: {
+            Plots: {
+              resize: vi.fn(),
+            },
+          },
+        })),
+      },
+      writable: true,
+    });
+
+    renderComponentWithProviders(
+      <VectorWindow
+        onClose={vi.fn()}
+        vectorConfig={testVectorConfig}
+        vectorWindowRef={ref}
+      />
+    );
+
+    await user.click(
+      screen.getByRole('button', { name: 'Show Vector Controls' })
+    );
+    expect(
+      await screen.findByRole('button', { name: 'Hide Vector Controls' })
+    ).toBeVisible();
+
```

Comment:
Might be worth making the `vi.fn()` external to the `Object.defineProperty` portion so we can assert that `resize` has been called here?